lambdas/importer.ts

```diff
@@ -7,7 +7,7 @@ async function fetchEvents(meetupAccessToken: string) {
 	const meetupGraphQlEndpoint = 'https://api.meetup.com/gql';
 	const batchSize = 10; // Number of events to fetch in each batch
 
-	const GROUP_URLNAMES = ['open-sgf', 'springfield-women-in-tech', 'sgfdevs'];
+	const GROUP_URLNAMES = process.env['MEETUP_GROUP_URLNAMES']?.split(',');
```

Comment:
Honestly, it might be a good idea to automatically remove spaces from these values just in case someone else accidentally adds them;.

So something like `const GROUP_URLNAMES = (process.env.MEETUP_GROUP_URLNAMES?.split(',') ?? []).map((group => group.trim());`